pkg/action/diff.go

```diff
@@ -0,0 +1,65 @@
+package action
+
+import (
+	"errors"
+	"os"
+
+	"github.com/helmwave/helmwave/pkg/plan"
+	"github.com/urfave/cli/v2"
+)
+
+type Diff struct {
+	plandir1, plandir2 string
+}
+
+func (d *Diff) Run() error {
+	if d.plandir1 == d.plandir2 {
+		return errors.New("plan1 and plan2 are the same")
+	}
+
+
+	plan1 := plan.New(d.plandir1)
+	if err := plan1.Import(); err != nil {
+		return err
+	}
+	if ok := plan1.IsManifestExist(); !ok {
+		return os.ErrNotExist
```

Comment:
I think this should be wrapped into another error defining which plan caused this error